src/Options/Applicative/Builder/Internal.hs

```diff
@@ -181,5 +181,7 @@ mkProps (DefaultProp def sdef) g = props
       { propShowDefault = sdef <*> def }
 
 -- | Hide this option from the help text
+--
+-- Use 'hidden' to hide the option from the brief description too.
```

Comment:
This isn't quite right. Visibility is used as an ordered enumeration, so it's more like:

"use 'hidden' to only hide from the brief description."

Reply:
https://github.com/pcapriotti/optparse-applicative/blob/6ff428bd332af7458af93b63a523f40284049453/src/Options/Applicative/Types.hs#L140-L144

Reply:
Ah, thanks for clearing that up!